massa-bootstrap/src/server.rs

```diff
@@ -151,92 +152,98 @@ impl BootstrapServer {
                 }
 
                 // listener
```

Comment:
I don't understand the changes in this file. Intuitively I expect changes in this PR to be limited to the binder files, config, and maybe some tests

Reply:
I'll do some changes appart for the bootstrap who use a local duplex

Reply:
We should discuss those changes, I didn't catch what they are doing

Reply:
there still seem to be changes not restricted to the binders. What are they doing ?